src/payment/models/crypto-input/crypto-input.service.ts

```diff
@@ -253,24 +259,59 @@ export class CryptoInputService {
     });
   }
 
-  private async saveAndForward(input: CryptoInput): Promise<void> {
+  private async testCompositeSwaps(
```

Comment:
I would rename this method to something like `getReferenceAmounts()`. USDT, USDC and BTC are reference assets. These are the ones we (virtually) exchange on Kraken.

Reply:
done